src/site/stages/build/drupal/page.js

```diff
@@ -240,6 +249,20 @@ function compilePage(page, contentData) {
       );
       break;
     default:
+      // Get the right benefits hub sidebar
```

Comment:
Might make sense to pull this block into its own function. I think keeping this switch statement as concise as we can makes it easier to understand.